internal/storage/postgres/seed/test/demo_data/99_demo_data.sql

```diff
@@ -88,15 +87,15 @@ VALUES
 
 INSERT INTO sensor_data (sensor_id, data)
 VALUES 
-  (1, '{"temperature": 20.0, "humidity": 0.5, "moisture": 0.75}'),
-  (1, '{"temperature": 21.0, "humidity": 0.6, "moisture": 0.5}'),
-  (1, '{"temperature": 22.0, "humidity": 0.7, "moisture": 0.25}'),
-  (2, '{"temperature": 20.0, "humidity": 0.5, "moisture": 0.75}'),
-  (2, '{"temperature": 21.0, "humidity": 0.6, "moisture": 0.5}'),
-  (2, '{"temperature": 22.0, "humidity": 0.7, "moisture": 0.25}'),
-  (3, '{"temperature": 20.0, "humidity": 0.5, "moisture": 0.75}'),
-  (3, '{"temperature": 21.0, "humidity": 0.6, "moisture": 0.5}'),
-  (3, '{"temperature": 22.0, "humidity": 0.7, "moisture": 0.25}');
+  ('sensor-1', '{"temperature": 20.0, "humidity": 0.5, "moisture": 0.75}'),
+  ('sensor-1', '{"temperature": 21.0, "humidity": 0.6, "moisture": 0.5}'),
+  ('sensor-1', '{"temperature": 22.0, "humidity": 0.7, "moisture": 0.25}'),
+  ('sensor-2', '{"temperature": 20.0, "humidity": 0.5, "moisture": 0.75}'),
+  ('sensor-2', '{"temperature": 21.0, "humidity": 0.6, "moisture": 0.5}'),
+  ('sensor-2', '{"temperature": 22.0, "humidity": 0.7, "moisture": 0.25}'),
+  ('sensor-3', '{"temperature": 20.0, "humidity": 0.5, "moisture": 0.75}'),
+  ('sensor-3', '{"temperature": 21.0, "humidity": 0.6, "moisture": 0.5}'),
+  ('sensor-3', '{"temperature": 22.0, "humidity": 0.7, "moisture": 0.25}');
```

Comment:
This is the old structure. Or is the test using this data? If so, we should update the test

Reply:
Please take a look at the latest update of the file: 
https://github.com/green-ecolution/green-ecolution-backend/blob/feature/edit-database-for-sensors/internal/storage/postgres/seed/99_demo_data.sql

Reply:
Yes, and now? This is a different file for a different use case. This is for testing. But I don't see this file being used at all in the project, so please delete it.

Reply:
Yes, you are right 👍